build/packer/pmm.json

```diff
@@ -77,16 +77,17 @@
         "--version", "{{isotime \"2006-01-02\"}}",
         "--description", "Percona Monitoring and Management (PMM) is an open-source platform for managing and monitoring MySQL, PostgreSQL and MongoDB"
       ],
-      "format": "ovf",
+      "format": "ova",
       "guest_additions_mode": "disable",
       "headless": true,
-      "output_directory": "pmm-virtualbox-ovf",
-      "shutdown_command": "rm -rf ~/.ssh/authorized_keys; cat /dev/zero > zero.fill; sync; sleep 1; sync; rm -f zero.fill; sudo shutdown -P now",
```

Comment:
can we have any other keys except packer one as it seems like packer removes it's own keys only [ssh_clear_authorized_keys](https://developer.hashicorp.com/packer/docs/communicators/ssh#ssh_clear_authorized_keys) (bool) - If true, Packer will attempt to remove its temporary key from ~/.ssh/authorized_keys and /root/.ssh/authorized_keys. This is a mostly cosmetic option, since Packer will delete the temporary private key from the host system regardless of whether this is set to true (unless the user has set the -debug flag). Defaults to "false"; currently only works on guests with sed installed.